Makefile.Crossdock.mk

```
@@ -32,7 +32,7 @@ build-crossdock-fresh: build-crossdock-linux
 	make crossdock-fresh
 
 .PHONY: crossdock-docker-images-jaeger-backend
-crossdock-docker-images-jaeger-backend: create-baseimg create-debugimg
+crossdock-docker-images-jaeger-backend: crossdock-create-baseimg crossdock-create-debugimg
```

Have you tried just overriding the variable?
```suggestion
crossdock-docker-images-jaeger-backend: PLATFORMS=linux/amd64
crossdock-docker-images-jaeger-backend: create-baseimg create-debugimg
```

---

@yurishkuro Yeah! It's working!  There is a lot to learn and explore under your guidance in this distributed platform :)